src/borrower/deployer.sol

```diff
@@ -186,9 +193,14 @@ contract BorrowerDeployer {
         AuthLike(shelf).rely(root);
     }
 
-    function deployCeiling() public {
+    function deployCeiling(bytes32 ceiling_) public {
```

Comment:
Our deployer should have any optional parameter.

It would break the entire concept